### PR TITLE
Acknowledgements: Added ELLIIT as funding source

### PR DESCRIPTION
**Pull request by ResearchDaniel**

### CODE REVIEW

**Review by martinfalk — CHANGES_REQUESTED** [review]
The logos shipped with Inviwo usually come only in sizes 200x200 and 600x600 (see `docs/images/`). The elliit logo is 1579x1578 pixels.

**Comment by martinfalk** [comment]
Good catch!

**Comment by ResearchDaniel** [comment]
> The logos shipped with Inviwo usually come only in sizes 200x200 and 600x600 (see `docs/images/`). The elliit logo is 1579x1578 pixels.

Fixed

**Review by martinfalk — DISMISSED** [review]

**Review by petersteneteg — CHANGES_REQUESTED** [review]
What about the about dialog?

**Comment by ResearchDaniel** [comment]
> What about the about dialog?

Fixed

**Comment by ResearchDaniel** [comment]
Seems like there are some issues with the hdf repository:
fatal: repository 'https://github.com/live-clones/hdf5.git/' not found

File: src/qt/editor/inviwoaboutwindow.cpp [hunk 1]
```diff
@@ -186,6 +187,9 @@ InviwoAboutWindow::InviwoAboutWindow(InviwoMainWindow* mainwindow)
 
         auto uulm = p.append("a", "", {{"href", "http://www.uni-ulm.de/en/"}});
         uulm.append("img", "", makeImg(":/images/uulm.png", 50));
+
+        auto elliit = p.append("a", "", {{"href", "https://old.liu.se/elliit?l=en"}});
+        elliit.append("img", "", makeImg(":/images/elliit.png", 50));
```
Comment (petersteneteg) [inline src/qt/editor/inviwoaboutwindow.cpp:192]: Have you tested this? I don't see any adding to the .rc file for the image?

**Review by petersteneteg — CHANGES_REQUESTED** [review]

**Comment by petersteneteg** [comment]
you have to update your submodules, we use the official HDFGroup repo now for hdf5

File: src/qt/editor/inviwoaboutwindow.cpp (hunk 1 above)
Comment (ResearchDaniel) [reply src/qt/editor/inviwoaboutwindow.cpp:192]: Ah, right. Added it and tested it. Changed to transparent background to be consistent with the others

**Review by petersteneteg — DISMISSED** [review]

**Review by petersteneteg — APPROVED** [review]

**Comment by ResearchDaniel** [comment]
> you have to update your submodules, we use the official HDFGroup repo now for hdf5

I have rebased onto master and done a clean build on jenkins. Still same error. It seems to run the module sync/update on jenkins but it still produces an error for warn. Any ideas?

**Comment by petersteneteg** [comment]
But the merge one works right? we only care about that one... probably have to delete the source dir in jenkins to properly get it to pull the fresh submodule. You can just go ahead and merge I think

**Pull request merged**